### ISSUE TITLE
scss: multiple params in :not() breaks output

### ISSUE THREAD

**Issue by epelc** [prettydiff/prettydiff#304]
Multiple paramaters in a `:not()` breaks the output. It seems to be due to the rules that move comma separate selectors onto multiple lines. I'd also guess this is affecting things besides `:not()`.

given
```
a:not(md-icon, .no-margin-bottom) {
    margin-bottom: 10px;
}
```

you get
```
.no-margin-bottom),
a:not(md-icon {
    margin-bottom: 10px;
}
```

![image](https://cloud.githubusercontent.com/assets/5204642/15186497/473a1054-176b-11e6-8d1e-c27cb79c65fa.png)

I ran into this while using atom-beautify but it is an issue in prettydiff. I also have the same issue(slightly different output but it's still broken and unusable) while using CSScomb in atom-beatuify.

Also see #23 and the chain of related issues

**Comment by prettydiff** [comment]
Sorry for the long delay.  I have been away at a military school.  I will look into resolving this issue when I return home this weekend.

**Comment by prettydiff** [comment]
Fixed in branch 2.0.4.
